account-kit/react/src/components/auth/card/main.tsx

```diff
@@ -25,9 +25,12 @@ export const MainAuthContent = ({ authStep }: MainAuthContentProps) => {
         return (
           <Fragment key={`auth-section-fragment-${idx}`}>
             <AuthSection key={`auth-section-${idx}`} authTypes={section} />
-            {idx !== sections.length - 1 ? (
+            {idx === sections.length - 1 ||
+            (sections[idx][0].type === "passkey" &&
+              idx + 1 < sections.length &&
+              sections[idx + 1][0].type === "social") ? null : (
               <Divider key={`divider-${idx}`} />
-            ) : null}
+            )}
```

Comment:
what's this doing exactly? Before it was adding a divider after each section which is the intended behaviour. If you want social auth to be in the same section as passkey, then you would jsut add the `{type: social ...}` section to the same array as the passkey one